lib/mentionsPlugin.js

```diff
@@ -30,13 +30,19 @@ function parseMentions (state, silent) {
     return false;
   }
 
-  const text = state.src.slice(state.pos);
-  const match = text.match(mentionRegex);
-
-  if (!match || partOfLinkOrEmail(state.src.substr(0, state.pos), text, state.md)) {
+  const textUntilAt = state.src.slice(0, state.pos);
+  const textAfterAt = state.src.slice(state.pos);
+  
```

Comment:
eslint is showing an extra trailing space at this line 
<img width="752" alt="Schermata 2020-08-25 alle 17 23 50" src="https://user-images.githubusercontent.com/589911/91193780-c97dd900-e6f7-11ea-80a1-142bb27ac355.png">